app/assets/stylesheets/pages/solution-page.scss

```diff
@@ -362,20 +359,36 @@ body:not(.prism-dark) #teams-my-solution-page {
     }
 }
 
-body.prism-dark {
+body:not(.prism-dark) {
```

Comment:
I'm not sure if this is the right place for this. I think we should extract a separate component for the prism box soon and handle themes in there. I'm okay with this for now as it is how we have things right now.

Reply:
> I'm not sure if this is the right place for this. 

Agree. But I think this then becomes the only file we use it in (We already use it lots in this file already)

>  think we should extract a separate component for the prism box soon and handle themes in there

I couldn't agree more.